hook-emulator/package.json

```diff
@@ -2,12 +2,14 @@
   "name": "hook-emulator",
   "scripts": {
     "lint": "eslint ./hook-emulator.js",
-    "build:wrapper": "gcc -m32 ./c_wrappers/main.c ../src/evernode.c ../lib/emulatorapi.c -o ./c_wrappers/hook-wrapper",
-    "build": "npm run build:wrapper && npm run lint && ncc build hook-emulator.js --minify -o dist",
-    "start": "npm run build:wrapper && node hook-emulator.js"
+    "build:wrapper": "gcc -m32 ./c-wrappers/main.c ../src/evernode.c ../lib/emulatorapi.c -o ./dist/hook-wrapper",
+    "build:accsetup": "npm run --prefix ../utils/account-setup build && cp -r ../utils/account-setup/dist dist/account-setup",
+    "build:self": "npm run lint && ncc build hook-emulator.js --minify -o dist",
+    "build": "npm run build:self && npm run build:wrapper && npm run build:accsetup",
+    "bundle": "rm -r dist && npm run build && cp -r dist emulator && cp -r sec emulator/ && cp run-emulator.sh emulator/ && tar cfz dist/emulator.tar.gz emulator && rm -r emulator"
```

Comment:
`rm -rf dist ...`
If not the command will throw an error when the dist folder does not exist which could happen in a first time run.